assets/i18n/hi.json

```diff
@@ -0,0 +1,276 @@
+{
+  "I18N_ABOUT_PAGE_TITLE": "हमारे बारे में - Oppia",
+  "I18N_ACTION_BROWSE_EXPLORATIONS": "हमारे एक्सप्लोरेशन्स ब्राउज़ करे",
+  "I18N_ACTION_CREATE_EXPLORATION": "अपने एक्सप्लोरेशन बनाये",
+  "I18N_CREATE_ACTIVITY_QUESTION": "आप क्या बनाना चाहेंगे ?",
+  "I18N_CREATE_ACTIVITY_TITLE": "एक्टिविटी बनाये",
+  "I18N_CREATE_COLLECTION": "कलेक्शन बनाये",
+  "I18N_CREATE_EXPLORATION": "एक्सप्लोरेशन बनाये",
+  "I18N_CREATE_EXPLORATION_CREATE": "सर्जन करना",
+  "I18N_CREATE_EXPLORATION_QUESTION": "क्या आप एक्सप्लोरेशन बनाना चाहेंगे ?",
+  "I18N_CREATE_EXPLORATION_TITLE": "एक्सप्लोरेशन बनाये",
+  "I18N_CREATE_EXPLORATION_UPLOAD": "अपलोड",
+  "I18N_CREATE_NO_THANKS": "नहीं, धन्यवाद",
+  "I18N_CREATE_YES_PLEASE": "हाँ!",
+  "I18N_DASHBOARD_OPEN_FEEDBACK": "फीडबैक देखे",
+  "I18N_DASHBOARD_MY_COLLECTIONS": "मेरे कलेक्शन्स",
+  "I18N_DASHBOARD_MY_EXPLORATIONS": "मेरे एक्सप्लोरेशन्स",
+  "I18N_DIRECTIVES_DRAG_IMAGE_HERE": "यहाँ छवि को खींचें",
+  "I18N_DIRECTIVES_OR_SELECT_FILE": "या एक छवि फ़ाइल को चुने:",
+  "I18N_ERROR_DISABLED_EXPLORATION": "निर्योग्य एक्सप्लोरेशन ",
+  "I18N_ERROR_DISABLED_EXPLORATION_MESSAGE": "माफ़ कीजिये,ये एक निर्योग्य एक्सप्लोरेशन है। बाद में पुन: प्रयास करें ।",
+  "I18N_ERROR_DISABLED_EXPLORATION_PAGE_TITLE": "निर्योग्य एक्सप्लोरेशन - Oppia",
+  "I18N_ERROR_HEADER_400": "Error 400",
+  "I18N_ERROR_HEADER_401": "Error 401",
+  "I18N_ERROR_HEADER_404": "Error 404",
+  "I18N_ERROR_HEADER_500": "Error 500",
+  "I18N_ERROR_MESSAGE_400": "कभी कभी मशीन्स मनुष्यो को नहीं समझ सकते। यह उस काल में से एक है।",
```

Comment:
"कभी कभी मशीन्स मनुष्य को नहीं समझ सकती | यह उस समय में से एक है "
How about above line.
"काल" or "समय" are almost same but what I think is that "सकते" should be replaced with "सकती"